roles/ogm/tasks/script_and_cron.yml

```diff
@@ -34,14 +34,14 @@
      hour: '02'
      minute: '31'
      user: 'umcg-ogm-dm'
-     job: /bin/bash -c '~/make_backup.sh'
+     job: /bin/bash -c '~/make_backup.sh >> ~/make_backup.sh.log'
```

Comment:
When we append to a log file in a home dir is there some logrotation / cleanup in place to prevent this from eventually run out of disk space on the machine that executes this cron job?

Reply:
There are Maximim 10 lines of log per BAS server. There is no need to be scared of logs exploding. Currently there is 16k of logs for the last week or two :)